internal/sops/awskms/keysource_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2022 The Flux authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package awskms
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	. "github.com/onsi/gomega"
+)
+
+func TestCreds_ApplyToMasterKey(t *testing.T) {
```

Comment:
Instead of copying the tests from upstream, please rewrite them to get the same coverage as the other keyservice implementations, while also [including `_Compat` tests](https://github.com/fluxcd/kustomize-controller/pull/635#discussion_r861344121).